crates/proof-of-sql/src/sql/proof_exprs/literal_expr.rs

```diff
@@ -57,11 +57,11 @@ impl<C: Commitment> ProofExpr<C> for LiteralExpr<C::Scalar> {
     #[tracing::instrument(name = "LiteralExpr::prover_evaluate", level = "debug", skip_all)]
     fn prover_evaluate<'a>(
         &self,
-        builder: &mut FinalRoundBuilder<'a, C::Scalar>,
+        table_length: usize,
+        _builder: &mut FinalRoundBuilder<'a, C::Scalar>,
         alloc: &'a Bump,
         _accessor: &'a dyn DataAccessor<C::Scalar>,
     ) -> Column<'a, C::Scalar> {
-        let table_length = builder.table_length();
```

Comment:
Should we remove `FinalRoundBuilder::table_length` since it no longer should be used?

Reply:
Yes. That's overdue. In fact we really need to have one `FinalRoundBuilder` per `ProofPlan`.